CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!-- markdownlint-disable MD041 -->
 ## 0.17.0 (Unreleased)
 
+BUG FIXES:
```

Comment:
Looks like we have two lots of bug fixes for this release, can you add lower down. Thanks.